services/QuillLMS/engines/evidence/lib/evidence/evidence/check/regex_post_topic.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Evidence
+  class Check::RegexPostTopic < Check::Base
```

Comment:
Note, the three `Check::Regex...` classes are all the same except for they pass in a different rule type. I could have subclassed them, but I wasn't convinced that would be cleaner given these are such small classes.